src/components/common/Tree/CustomTreeItem.js

```diff
@@ -52,25 +58,30 @@ const CustomTreeItem = ({ itemId, expandedItems = [], selectedId }) => {
     }
     const filteredChildren = children?.filter(
       (child, idx) =>
-        !isHidden(childrenTags?.get(idx)) && child.type === ITEM_TYPES.FOLDER,
+        !isHidden(childrenTags?.get(idx)) &&
+        [ITEM_TYPES.FOLDER, ITEM_TYPES.SHORTCUT].includes(child.type),
```

Comment:
Maybe we can factor out the array to avoid re-allocating it on each call. Though it's a very small nitpick.